src/SignalR/clients/java/signalr/test/src/main/java/com/microsoft/signalr/LongPollingTransportTest.java

```diff
@@ -125,6 +125,7 @@ public void CanSetAndTriggerOnReceive() {
         assertTrue(onReceivedRan.get());
     }
 
+    @Ignore("https://github.com/dotnet/aspnetcore/issues/34563")
```

Comment:
```suggestion
    @Disabled
```

Reply:
And add `import org.junit.jupiter.api.Disabled;` near the top of the file

Reply:
Why not just add the `import org.junit.Ignore;` I missed❔

Reply:
If it works, sure